graphql/queryResolvers/moduleCrud.ts

```diff
@@ -0,0 +1,29 @@
+import type { MutationDeleteModuleArgs, MutationAddModuleArgs } from '..'
+import prisma from '../../prisma'
+
+export const modules = async () => {
+  return prisma.module.findMany({
+    include: {
+      author: true
+    }
+  })
+}
+
+export const addModule = async (_parent: void, args: MutationAddModuleArgs) => {
+  const { authorId, content, lessonId, name } = args
+  if (!authorId || !content || !name || !lessonId) {
```

Comment:
Are we only trying to check if all arguments are present, or instead are we checking if the arguments are not the correct value? For example, if I passed the `content` value as empty string(`''`), is that considered a missing parameter?

Reply:
If you are only checking if all required arguments are present, you don't have to do that. Graphql takes care of that for you already -- if you marked in the typedefs that arguments are required for this resolver, then the graphql request will already return an error if a person does not include required arguments in their request.